source/codegen/metadata/nidmm/functions.py

```diff
@@ -1624,7 +1624,7 @@
     },
     "InitExtCal": {
         "init_method" : True,
-        "codegen_method" : None,
+        "custom_close" : "CloseExtCal(id,NIDMM_EXTCAL_ACTION_ABORT)",
```

Comment:
Put a space after the `,`:
```suggestion
        "custom_close" : "CloseExtCal(id, NIDMM_EXTCAL_ACTION_ABORT)",
```

Reply:
Done